zipline/data/minute_bars.py

```diff
@@ -415,11 +413,49 @@ def write(self, sid, df):
                 volume : float64|int64
             index : DatetimeIndex of market minutes.
         """
+        cols = {
+            'open': df.open.values,
+            'high': df.high.values,
+            'low': df.low.values,
+            'close': df.close.values,
+            'volume': df.volume.values,
+        }
+        dts = df.index.values
```

Comment:
When you say "internal code", do you mean internal to a loop or code in an internal repo?  Wondering if zipline algos will need to call this function in a tight loop, or if it's just a one-time setup.